src/main/java/frc/robot/subsystems/drivetrain/commands/DriveSlowAccel.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+
+import java.util.function.DoubleSupplier;
+
+public class DriveSlowAccel extends HolonomicDrive {
```

Comment:
Why do you need this command? (we already use the Rate limiter, which should do the same)